sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/json/JsonFileFormat.scala

```diff
@@ -95,6 +95,7 @@ case class JsonFileFormat() extends TextBasedFileFormat with DataSourceRegister
       options: Map[String, String],
       hadoopConf: Configuration): PartitionedFile => Iterator[InternalRow] = {
     val broadcastedHadoopConf =
+      SerializableConfiguration.broadcast(sparkSession.sparkContext, hadoopConf)
```

Comment:
@yaooqinn @dongjoon-hyun  i wonder if the next line can be removed (as its return value is no longer used, so looks like its a wasted side-effect)?  I made a pr for it:  https://github.com/apache/spark/pull/50874

Reply:
Oh, right. Thank you, @szehon-ho .